album.html

```diff
@@ -1,42 +1,43 @@
-         <nav class="navbar"> <!-- nav bar -->
-             <a href="index.html"  class="logo">
-                 <img src="assets/images/bloc_jams_logo.png" alt="bloc jams logo">
-             </a>
-             <div class="links-container">
-                 <a href="collection.html" class="navbar-link">collection</a>
-             </div>
-         </nav>
-         
-         <main class="album-view container narrow">
-             <section class="clearfix">
-                 <div class="column half">
-                     <img src="assets/images/album_covers/01.png" class="album-cover-art">
-                 </div>
-                 <div class="album-view-details column half">
-                     <h2 class="album-view-title">The Colors</h2>
-                     <h3 class="album-view-artist">Pablo Picasso</h3>
-                     <h5 class="album-view-release-info">1909 Spanish Records</h5>
-                 </div>
-             </section>
-             <table class="album-view-song-list">
-             </table>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Bloc Jams</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Open+Sans:400,800,600,700,300">
+        <link rel="stylesheet" type="text/css" href="http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
+        <link rel="stylesheet" type="text/css" href="styles/normalize.css">
+        <link rel="stylesheet" type="text/css" href="styles/main.css">
+        <link rel="stylesheet" type="text/css" href="styles/album.css">
+        <link rel="stylesheet" type="text/css" href="styles/player_bar.css">
+    </head>
+    <body class="album">
+        <nav class="navbar">
+            <a href="index.html" class="logo">
+                <img src="assets/images/bloc_jams_logo.png" alt="bloc jams logo">
+            </a>
+            <div class="links-container">
+                <a href="collection.html" class="navbar-link">Collection</a>
+            </div>
+        </nav>
+        
+        <main class="album-view container narrow">
+            <section class="clearfix">
+                <div class="column half">
+                    <img src="assets/images/album_covers/01.png" class="album-cover-art">
+                </div>
+                <div class="album-view-details column half">
+                    <h2 class="album-view-title">The Colors</h2>
+                    <h3 class="album-view-artist">Pablo Picasso</h3>
+                    <h5 class="album-view-release-info">1909 Spanish Records</h5>
+                </div>
+            </section>
+            <table class="album-view-song-list">
+            </table>
         </main>
+        
         <section class="player-bar">
             <div class="container">
-                <div class="player-bar-control-group left-controls">
+                <div class="control-group main-controls">
```

Comment:
We've changed some class names here. `player-bar-control-group` is now just `control-group` and `left-controls` and `right-controls` have been replaces with `main-controls` and `volume`, respectively.